From f34e1d6b0ecf907066b6083e70d5eac84c2ba2f5 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 20 Aug 2013 15:09:33 +0200
Subject: [PATCH] network: do not depend on ifcfg

rather let ifcfg return 0 if /etc/sysconfig/network-scripts exists
---
 modules.d/40network/module-setup.sh | 1 -
 modules.d/45ifcfg/module-setup.sh   | 1 +
 2 files changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/40network/module-setup.sh b/modules.d/40network/module-setup.sh
index a5f635d..464e0cb 100755
--- a/modules.d/40network/module-setup.sh
+++ b/modules.d/40network/module-setup.sh
@@ -16,7 +16,6 @@ check() {
 }
 
 depends() {
-    [ -d /etc/sysconfig/network-scripts/ ] && echo ifcfg
     return 0
 }
 
diff --git a/modules.d/45ifcfg/module-setup.sh b/modules.d/45ifcfg/module-setup.sh
index 3cd0c48..ce53e5e 100755
--- a/modules.d/45ifcfg/module-setup.sh
+++ b/modules.d/45ifcfg/module-setup.sh
@@ -3,6 +3,7 @@
 # ex: ts=8 sw=4 sts=4 et filetype=sh
 
 check() {
+    [[ -d /etc/sysconfig/network-scripts ]] && return 0
     return 255
 }
 
